### PR TITLE
Update operating systems guidance

### PR DESCRIPTION
**Pull request by soupdragon99**

### CODE REVIEW

File: source/standards/operating-systems.html.md.erb [hunk 1]
```diff
@@ -1,34 +1,18 @@
...
+title: How to choose operating systems
 expires: 2018-01-31
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-When you operate a server, you need to choose an operating system for it.  This guidance makes recommendations for choosing an operating system for a Virtual Machine (VM).
+When choosing an operating system (OS) for your Virtual Machine (VM) we recommend using [Ubuntu Long Term Support (LTS)](https://www.ubuntu.com/download/desktop).
```
Comment (lapthorn) [inline source/standards/operating-systems.html.md.erb]: Surely https://www.ubuntu.com/download/server in the case of a server?

Comment (soupdragon99) [reply source/standards/operating-systems.html.md.erb]: Of course, thank you. URL now changed!

File: source/standards/operating-systems.html.md.erb [hunk 2]
```diff
@@ -1,34 +1,18 @@
 ---
-title: Operating Systems
+title: How to choose operating systems
 expires: 2018-01-31
```
Comment (tomnatt) [inline source/standards/operating-systems.html.md.erb]: Does this date need moving on now?

Comment (soupdragon99) [reply source/standards/operating-systems.html.md.erb]: Not at the moment, expiry dates were\are agreed by the tech forum. So this is just to tighten up language and ensure any content we publish externally matches GOV.UK style.

**Comment by soupdragon99** [comment]
Thanks for the review @tomnatt I'll submit this for 2i before we can publish.

**Comment by soupdragon99** [comment]
@tomnatt I have made a few changes to this guidance based on the 2i feedback, please review.

File: source/standards/operating-systems.html.md.erb [hunk 3]
```diff
@@ -1,34 +1,22 @@
...
+You must also make sure your OS is:
 
-### User needs
+* automatically updated for security issues
+* up to date with the latest OS releases
+* able to run on a local VM
 
-- the ability for people to move teams without learning a new OS or distribution
-- the ability to know that we are using a secure and maintained OS?
+If you need a newer kernel, or features not yet available in Ubuntu LTS then you can use a standard Ubuntu release.
```
[the remark this answers is not in the document]
Comment (soupdragon99) [reply source/standards/operating-systems.html.md.erb]: @tomnatt I have reworded this now, what do you think?

[the remark this answers is not in the document]
Comment (tomnatt) [reply source/standards/operating-systems.html.md.erb]: Much clearer - thank you 👍

**Pull request merged**